### ISSUE TITLE
Class vs ID usage

### ISSUE THREAD

**Issue by jericahuang** [doctorduudu/teacozy#4]
I notice you've used mostly all classes for your styling, and there's nothing wrong about that, but I would suggest looking into using IDs. Typically items that appear only once in an HTML page are assigned IDs. This doesn't make a difference for styling but it will for JavaScript! It will also make the navbar links work as I mentioned in another issue.

See this resource for more: https://css-tricks.com/the-difference-between-id-and-class/
